packages/tools/devtools/devtools-view/src/components/TelemetryView.tsx

```diff
@@ -447,6 +510,12 @@ function FilteredTelemetryView(props: FilteredTelemetryViewProps): React.ReactEl
 					size="extra-small"
 					resizableColumns
 					selectionMode="single"
+					subtleSelection
+					onSelectionChange={(e, data): void => {
+						// Set the index to the appropriate row index in the table.
+						setIndex(Number([...data.selectedItems][0]));
+					}}
+					selectedItems={[index !== undefined && index > 0 ? index : 0]}
```

Comment:
Doesn't this result in the 0th ID item being selected when we don't have a selection specified? Should this not be an empty array in that case?

Reply:
Nah, I tried that and you need to pass in some index because selectedItems points some TableRowId.